### ISSUE TITLE
[Feature] Add GPT-4 Mobile API hack for unlimited GPT-4 access

### ISSUE THREAD

**Issue by lishuhang** [lencx/ChatGPT#936]
### Feature description

There is a [tampermonkey script](https://greasyfork.org/zh-CN/scripts/467317-chatgpt%E5%BC%80%E5%90%AF%E4%B8%8D%E9%99%90%E6%AC%A1%E6%95%B0%E7%9A%84gpt4-mobile) that can transplant ChatGPT iOS client's API into web interface, therefore web users may toggle unlimited GPT-4 access, not restricted by 25 messages / hour limit.

I'll appreciate if you add the script's function into your client.

**Comment by lencx** [comment]
Please update the script, you can view the demo here: https://youtu.be/AZSe3sHH9_M

`Control Center -> Scripts -> chat.js -> Sync -> Restart App`

**Comment by MindMelodies** [comment]
Is there a plan to incorporate plugin capabilities into GPT-4-Mobile?

**Comment by 0xorio** [comment]
<img width="656" alt="CleanShot 2023-07-20 at 12 33 36@2x" src="https://github.com/lencx/ChatGPT/assets/71540271/f5c30bba-5d56-4704-9eee-978be548d7e3">

looks its already implemented in the latest version so it could be closed?

**Comment by skywinder** [comment]
<img width="656" alt="CleanShot 2023-07-20 at 12 33 36@2x" src="https://github.com/lencx/ChatGPT/assets/71540271/f5c30bba-5d56-4704-9eee-978be548d7e3">

looks its already implemented in the latest version so it could be closed?

**Comment by skywinder** [comment]
oops. but 3 hors ago looks mobile was removed again. is it stop working? https://github.com/lencx/ChatGPT/commit/119570c4d02c2f9469ee8ff0a200c8bf511bc11a @lencx ?

**Comment by lencx** [comment]
@skywinder Yes, we removed it because unlimited usage is no longer supported in Mobile mode. Do you still need this feature?

**Comment by Yizhan-Gu** [comment]
Any other ways of using gpt-4?
